### PR TITLE
Added Owlbot command

### LINKED ISSUE #1123: Owlbot Dictionary Command
`s!owlbot <word>` - Uses Owlbot to define an english word

https://owlbot.info/

### PR DESCRIPTION
**Pull request by upedd**
closes #1123

### CODE REVIEW

**Comment by upedd** [comment]
image

**Comment by almeidx** [comment]
Add the OWLBOT_KEY env variable to the Environment Variables wiki

**Comment by upedd** [comment]
Where is it?

**Comment by almeidx** [comment]
> Where is it?

https://github.com/SwitchbladeBot/switchblade/wiki/Environment-Variables

**Comment by upedd** [comment]
Done

**Review by almeidx — APPROVED** [review]

File: src/commands/misc/owlbot.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              value: `${d.definition}${d.example ? `\n"_${d.example}_"` : ''}`
+            })
+          )
+        )
+        .setColor(Constants.OWLBOT_COLOR)
+        .setFooter(
+          t('commands:owlbot.footer'),
+          'https://media.owlbot.info/dictionary/images/owlbot.png.400x400_q85_box-0,0,287,288_crop_detail.png'
+        )
+      if (data.pronunciation) embed.setDescription(data.pronunciation)
```
Comment (pedrofracassi) [inline src/commands/misc/owlbot.js:39]: Add these two slashes, they're important for written pronunciation :)

image

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

**Comment by upedd** [comment]
Fixed

**Review by almeidx — DISMISSED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]

**Review by jaxxibae — APPROVED** [review]
LGTM

**Pull request merged**